pkg/nsx/services/vpc/vpc.go

```diff
@@ -781,3 +808,44 @@ func (vpcService *VPCService) getLBProvider() string {
 	}
 	return LBProviderAVI
 }
+
+// ListNamespacesWithPreCreatedVPC returns a mapping of the pre-created VPC path and the Namespaces
+// which use the VPC. The key is the pre-created VPC path, the value is a slice of Namespaces that
+// use the VPC.
+func (service *VPCService) ListNamespacesWithPreCreatedVPC() map[string][]string {
```

Comment:
where is it used?

Reply:
My fault, originally I have added a separate goroutine to periodically read the pre-created VPC's settings and update to NetworkInfo CR, and I removed that part from this change. It seems I didn't remove code clean. Will remove it.